Gemfile.lock

```diff
@@ -315,76 +316,78 @@ GEM
       momentjs-rails (>= 2.8.1)
     brakeman (5.4.1)
     builder (3.2.4)
-    bundler-audit (0.9.0.1)
+    bundler-audit (0.9.1)
       bundler (>= 1.2.0, < 3)
       thor (~> 1.0)
     byebug (8.2.5)
-    chef-utils (18.1.29)
+    chef-utils (18.4.2)
       concurrent-ruby
-    coderay (1.1.1)
+    coderay (1.1.3)
     commonmarker (0.23.10)
-    concurrent-ruby (1.2.2)
-    connection_pool (2.2.1)
-    crack (0.4.3)
-      safe_yaml (~> 1.0.0)
+    concurrent-ruby (1.2.3)
+    connection_pool (2.4.1)
+    crack (0.4.6)
```

Comment:
### [crack](https://github.com/jnunemaker/crack)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 0.4.3 → 0.4.6

[_[change-log](https://github.com/jnunemaker/crack/blob/master/History), [source-code](https://github.com/jnunemaker/crack), [gem-diff](https://my.diffend.io/gems/crack/0.4.3/0.4.6)_]





<details>
<summary>Commits</summary>

A change of **24** commits. See the full changes on [the compare page](https://github.com/jnunemaker/crack/compare/v0.4.3...v0.4.6).

These are the first 10 commits:
- (a7189fe) [remove safe_yaml](https://github.com/jnunemaker/crack/commit/a7189fef728ce522cb03f62b18848cfc50058245)
- (261962a) [Exclude tests (and fixtures) from bundled gem](https://github.com/jnunemaker/crack/commit/261962ae56f0c4216d0be6b8c65037ec5c853a70)
- (1183581) [Merge pull request #61 from greysteil/dont-bundle-tests](https://github.com/jnunemaker/crack/commit/1183581ded7d7ee6b427778eef430d3d10000e04)
- (4ecf9cf) [Merge pull request #59 from bf4/master](https://github.com/jnunemaker/crack/commit/4ecf9cf5f721dea61b8fa86280180142ead35312)
- (579acb3) [Add REXML to gem dependency](https://github.com/jnunemaker/crack/commit/579acb39c5597b5a4353181a5cf967c148625275)
- (288371d) [Bump version to 0.4.4; update history](https://github.com/jnunemaker/crack/commit/288371d0444ba64bde1b082f340fcc67eeb9e64e)
- (244855f) [GitHub is HTTPS by default](https://github.com/jnunemaker/crack/commit/244855f6ee77f85fd78a2e303de484fd17c60edf)
- (f61a62e) [Merge pull request #64 from amatsuda/https](https://github.com/jnunemaker/crack/commit/f61a62eb5b2ad877950831c5a69c6e230c6c73f2)
- (c61172b) [Merge pull request #62 from koic/add_rexml_to_gem_dependenci…](https://github.com/jnunemaker/crack/commit/c61172bf32e1769748fded156c2f2fc03dac69c1)
- (7ec6b55) [Bump to 0.4.5](https://github.com/jnunemaker/crack/commit/7ec6b55efd0055cc04e30be0fa828e0902c45c7b)


</details>